uberfire-layout-editor/uberfire-layout-editor-api/src/main/java/org/uberfire/ext/layout/editor/api/editor/LayoutTemplate.java

```diff
@@ -8,32 +8,32 @@
 import org.jboss.errai.common.client.api.annotations.Portable;
 
 @Portable
-public class LayoutEditor {
+public class LayoutTemplate {
```

Comment:
Could you please add final static string with template version? This would be useful to write the migration tools for your old perspectives.